craft_parts/executor/organize.py

```diff
@@ -57,14 +58,16 @@ def organize_files(
         the default partition.
     """
     for key in sorted(file_map, key=lambda x: ["*" in x, x]):
-        src_partition, src_inner_path = path_utils.get_partition_and_path(key)
+        src_partition, src_inner_path = path_utils.get_partition_and_path(
+            key, default_partition
+        )
 
-        if src_partition and src_partition != "default":
+        if src_partition and src_partition != default_partition:
```

Comment:
The `default` alias should also be valid when organizing. If literal `default` is found, we could do the internal logic using the default partition name.

Reply:
I wondered about that, it could be nice but also confusing for our users to still use `(default)` in their project file when they expect content to come from `(foo)` (the "renamed" `default`). I will implement what you suggest and I think we can discuss it with Gustavo when playing with the prototype.